src/data/finsupp/witnesses.lean

```diff
@@ -0,0 +1,116 @@
+/-
+Copyright (c) 2022 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+import data.finsupp.basic
+
+/-!
+#  `diff` for finitely supported functions
+
+Let `α N` be two Types, and assume that `N` has a `0` and let `f g : α →₀ N` be finitely supported
+functions.
+
+##  Main definition
+
+*  `finsupp.diff f g : finset α`, the finite subset of `α` where `f` and `g` differ,
```

Comment:
nit:
```suggestion
## Main definition

* `finsupp.diff f g : finset α`, the finite subset of `α` where `f` and `g` differ,
```

Reply:
You hit exactly the one line that I had changed on my local copy!  :upside_down_face: 
(I had changed the trailing comma to a period.)